plugins/woocommerce/includes/wc-conditional-functions.php

```diff
@@ -281,6 +295,18 @@ function is_lost_password_page() {
 	}
 }
 
+if ( ! function_exists( 'is_admin_settings_page' ) ) {
+
+	/**
+	 * is_admin_settings_page - Returns true when viewing the admin settings page.
+	 *
+	 * @return bool
+	 */
+	function is_admin_settings_page() {
+		return is_admin() && isset( $_GET['page'] ) && 'wc-settings' === $_GET['page']; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Micro-optimization suggession: move `is_admin()` to the last position in this expression. Function calls are more expensive than `isset` and identity checks.